### PR TITLE
Nu 1080/remove markdown stylus

### PR DESCRIPTION
**Pull request by bohdanprog**
## Checklist before merge
- [x] Related issue ID is placed at the beginning of PR title in \[brackets\] (can be GH issue or Nu Jira issue)
- [x] Code is cleaned from temporary changes and commented out lines
- [x] Code formatting checked
- [ ] Parts of the code that are not easy to understand are documented in the code
- [ ] Changes are covered by automated tests
- [ ] Showcase in dev-application.conf added to demonstrate the feature
- [ ] Documentation added or updated
- [ ] Added entry in _Changelog.md_ describing the change from the perspective of a public distribution user
- [ ] Added _MigrationGuide.md_ entry in the appropriate subcategory if introducing a breaking change
- [x] Verify that PR will be squashed during merge

### CODE REVIEW

File: designer/client/src/actions/nk/calculateProcessAfterChange.ts [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-import { mapProcessWithNewNode, replaceNodeOutputEdges } from "../../components/graph/GraphUtils";
-import { alignFragmentWithSchema } from "../../components/graph/FragmentSchemaAligner";
-import { Edge, NodeType, Process } from "../../types";
+import { mapProcessWithNewNode, replaceNodeOutputEdges } from "../../components/graph/utils/graphUtils";
+import { alignFragmentWithSchema } from "../../components/graph/utils/fragmentSchemaAligner";
+import { Edge, NodeType, Process, ProcessDefinitionData } from "../../types";
 import { ThunkAction } from "../reduxTypes";
 
-function alignFragmentsNodeWithSchema(process, processDefinitionData) {
+function alignFragmentsNodeWithSchema(process, processDefinitionData: ProcessDefinitionData) {
```
Comment (Dzuming) [inline designer/client/src/actions/nk/calculateProcessAfterChange.ts:9]: Can you also add type for `process` ;) ?

File: designer/client/src/components/comment/ProcessComments.tsx [hunk 2]
```diff
@@ -73,16 +73,16 @@ function ProcessComments(): JSX.Element {
...
                     </div>
                 ))}
-            </ul>
+            </ProcessCommentsList>
             {capabilities.write ? (
-                <div className="add-comment-panel">
+                <AddCommentPanel>
                     <CommentInput onChange={onInputChange.bind(this)} value={comment} />
-                    <NkButton type="button" className="add-comment" onClick={_addComment} disabled={pending || comment == ""}>
+                    <CommentNKButton type="button" onClick={_addComment} disabled={pending || comment == ""}>
```
Comment (Dzuming) [inline designer/client/src/components/comment/ProcessComments.tsx:80]: Do we need this NK in the name? Maybe `CommentButton` will be enough?

File: designer/client/src/components/comment/StyledComment.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
+import { styled } from "@mui/material";
+import { NkButton } from "../NkButton";
+import { variables } from "../../stylesheets/variables";
+
+export const ProcessCommentsWrapper = styled("div")`
```
Comment (Dzuming) [inline designer/client/src/components/comment/StyledComment.tsx:5]: Not sure if it's a good component for this comment, but there is a difference between bottom padding:

Localhost:

image

Staging:
image

File: designer/client/src/components/graph/types/index.ts [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import { ProcessCounts } from "../../../reducers/graph";
```
Comment (Dzuming) [inline designer/client/src/components/graph/types/index.ts:1]: Do we need this folder structure? `designer/client/src/components/graph/types/index.ts`? Maybe designer/client/src/components/graph/types.ts will be enough here?

File: designer/client/src/containers/errors/ErrorTemplate.tsx [hunk 5]
```diff
@@ -6,17 +7,50 @@ export interface ErrorTemplateProps {
...
+        margin: 45px auto;
+    }
+`;
+
 export function ErrorTemplate({ description = "", message = "", children }: PropsWithChildren<ErrorTemplateProps>): JSX.Element {
     const { t } = useTranslation();
 
     return (
-        <div className="error-template center-block">
+        <ErrorTemplateWrapper className="center-block">
```
Comment (Dzuming) [inline designer/client/src/containers/errors/ErrorTemplate.tsx:47]: Do we need this 'center-block' here?

File: designer/client/src/actions/nk/calculateProcessAfterChange.ts (hunk 1 above)
Comment (bohdanprog) [reply designer/client/src/actions/nk/calculateProcessAfterChange.ts:9]: When I tried add type in that function another function have a problem with type :)

File: designer/client/src/components/comment/ProcessComments.tsx (hunk 2 above)
Comment (bohdanprog) [reply designer/client/src/components/comment/ProcessComments.tsx:80]: done

File: designer/client/src/components/graph/types/index.ts (hunk 4 above)
Comment (bohdanprog) [reply designer/client/src/components/graph/types/index.ts:1]: I don't get you, why you want to change import type?

File: designer/client/src/containers/errors/ErrorTemplate.tsx (hunk 5 above)
Comment (bohdanprog) [reply designer/client/src/containers/errors/ErrorTemplate.tsx:47]: I didn't find that classname, so I decided to leave it there.

File: designer/client/src/components/comment/StyledComment.tsx (hunk 3 above)
Comment (bohdanprog) [reply designer/client/src/components/comment/StyledComment.tsx:5]: fix, and fix button size

File: designer/client/src/containers/errors/ErrorTemplate.tsx (hunk 5 above)
Comment (Dzuming) [reply designer/client/src/containers/errors/ErrorTemplate.tsx:47]: Me neither, that's why I'm wondering, do we need it and maybe we can just remove it?

File: designer/client/src/components/graph/types/index.ts (hunk 4 above)
Comment (Dzuming) [reply designer/client/src/components/graph/types/index.ts:1]: If I understand correctly this green icon means that this file was added in this mr, right?
image

so I suggest not making it like `types/index.ts` but rather as `types.ts`. 

Is it clear now?

File: designer/client/src/containers/errors/ErrorTemplate.tsx (hunk 5 above)
Comment (bohdanprog) [reply designer/client/src/containers/errors/ErrorTemplate.tsx:47]: I think we can :)

File: designer/client/src/components/graph/types/index.ts (hunk 4 above)
Comment (bohdanprog) [reply designer/client/src/components/graph/types/index.ts:1]: yes, I created that file,  I think you are talking about `import { GraphProps } from "./types/index";` 
I changed to that` import { GraphProps } from "./types";`

Comment (Dzuming) [reply designer/client/src/components/graph/types/index.ts:1]: No, I'm talking about moving this file to the folder up and renaming it to `types.ts`, I don't see any profits from keeping types in the separate folder and `index.ts` file in this case.

Comment (bohdanprog) [reply designer/client/src/components/graph/types/index.ts]: done

**Review by Dzuming — APPROVED** [review]

**Pull request merged**